docs/deploying.md

```diff
@@ -31,24 +31,33 @@ understanding in this area.
 
 ## Overview
 
-What federation really is are 4 or more Guardians independently running `fedimintd` node software.
-
-A `fedimintd` setup requires:
-
-* setting DNS domain
-* prunned `bitcoind` node
-* TLS termination software like `caddy` or `nginx`
-
+### Setting Up a Federation
+You can create a Fedimint in two ways:
+- Single Guardian Federation — A single guardian operating their own mint.
+- Group Federation — Multiple guardians running the mint together (minimum of 4 guardians).
+
+### Step By Step Ceremony
+1. Install & Launch `fedimintd`
+	Each participating guardian runs their own instance of fedimintd.
+	See below [self-hosted solutions](#self-hosted-solutions) or [hosted solutions](#hosted-solutions) for installation options.
+2. Set a Password
+	When starting fedimintd for the first time, each guardian must create a secure password.
+3. Generate Setup Code
+	After entering the password, a unique setup code is generated for each guardian.
+4. Exchange Setup Codes
+	All guardians must share their setup codes with each other (every guardian needs every other guardian’s code).
+5. Distributed Key Generation (DKG)
+	Once all setup codes are entered, the DKG process begins. This is where the federation’s cryptographic keys are jointly created.
+	This step may take some time while all guardians connect.
+6. Federation Complete!
+	Once DKG finishes, your federation is live. Your guardian dashboard will be available for monitoring. You now have a functioning Fedimint!
```

Comment:
Feels as if you had some fancier formatting in mind?

<img width="1196" height="355" alt="Screenshot 2025-08-28 at 18-55-52 fedimint_docs_deploying md at f4ff2698e71b5b7800450d0b64e30cdd048cb29f · fedimint_fedimint" src="https://github.com/user-attachments/assets/65383cd2-e29b-4c33-b573-e8f66979b486" />

Reply:
I didn't actually. These were mostly taken from the Umbrel PR. I just wanted step-by-step instructions for what a user should do once they're in the UI.